client/src/app/pages/todo/components/todo-form.component.ts

```
@@ -0,0 +1,82 @@
+  });
+
+  constructor(private formBuilder: FormBuilder) {
+    effect(() => {
+      this.todoForm.setValue({
+        text: this.initialState()?.text || '',
+      });
+    });
+  }
+
+  get text() {
+    return this.todoForm.get('text')!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)